packages/runtime-common/etc/test-fixtures.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-useless-escape */
```

Comment:
Disabling this because test fixtures have whitespace that prettier wants to fix. We don't want that because these are compared against generated code.